Example/android/app/build.gradle

```diff
@@ -216,7 +216,7 @@ dependencies {
 
 
     //clevertap
-    implementation 'com.clevertap.android:clevertap-android-sdk:4.6.6'
+    implementation 'com.clevertap.android:clevertap-android-sdk:4.7.2'
     implementation "com.clevertap.android:push-templates:1.0.5"
```

Comment:
@shivamsharma2710 update push templates sdk to v1.0.7

Reply:
this is done. please check.